packages/liquidsoap/liquidsoap.1.4.0/opam

```diff
@@ -0,0 +1,146 @@
+opam-version: "2.0"
+maintainer: "romain.beauxis@gmail.com"
+homepage: "https://github.com/savonet/liquidsoap"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+build: [
+  ["./bootstrap"] {dev}
+  ["./configure" "--prefix" prefix "--sbindir=%{lib}%/liquidsoap/sbin" "--libexecdir=%{lib}%/liquidsoap/libexec" "--sysconfdir=%{lib}%/liquidsoap/etc" "--sharedstatedir=%{lib}%/liquidsoap/com" "--localstatedir=%{lib}%/liquidsoap/var" "--libdir=%{lib}%/liquidsoap/lib" "--includedir=%{lib}%/liquidsoap/include" "--datarootdir=%{lib}%/liquidsoap/share" "--disable-graphics" "--with-user=dummy" "--with-group=dummy"]
+  [make "clean"] {dev}
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: [
```

Comment:
The `remove` section is unnecessary since opam 2.0

Reply:
Even with the binary and library install? Cool!

Reply:
yes